tox.ini

```diff
@@ -10,18 +10,14 @@ envlist = bandit, safety, black, black-check, isort, isort-check, fix-copyright,
 ; when running locally we don't want to fail for no good reason
 skip_missing_interpreters = true
 
-[testenv]
-basepython = python3.10
-whitelist_externals = /bin/sh
-passenv = *
-extras = all
+[base]
 deps =
```

Comment:
As in the `open-autonomy` please split this into deps for framework, tests and packages: https://github.com/valory-xyz/open-autonomy/pull/1411/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R13-R30

Reply:
Below we already define envs for `plugins`, `packages` and `framework` so use these imo and then add `tests`